independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+                      },
+                      "CustomTradeUnion": {
+                        "type": "string",
+                        "description": "CustomTradeUnion"
+                      },
+                      "CustomFields": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "Code": {
+                              "type": "string",
+                              "description": "Code"
+                            },
+                            "Label": {
+                              "type": "string",
+                              "description": "Label"
+                            },
+                            "Value": {
+                              "type": "string",
+                              "description": "Value"
+                            }
+                          }
+                        },
+                        "description": "CustomFields"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          }
+        }
+      },
+      "patch": {
+        "summary": "Update a single position record",
+        "description": "Update a single employee position record based on the specified EmployeePositionId.",
+        "operationId": "UpdateASinglePositionRecord",
+        "parameters": [
+          {
+            "name": "EmployeePositionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{EmployeePositionId}} The EmployeePositionId"
+          },
+          {
+            "name": "company-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{company-id}} The Company Id",
+            "x-ms-summary": "{{company-id}}"
+          },
+          {
+            "name": "CustomAuthHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Bearer token value (without the keyword Bearer)",
+            "x-ms-summary": "Bearer token"
+          },
+          {
+            "name": "CustomEnvironmentHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Variable that is either api (for PROD) or apistaging (for STAGING)",
+            "x-ms-summary": "Environment Variable"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "string"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {}
```

Comment:
@mint-virgil is it possible to remove the empty properties. if you are trying to use dynamic schema, please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/openapi-extensions#how-to-use-dynamic-schema).
Please fix all schemas in your connector swagger file.

Reply:
updated in latest commit